README.md

```diff
@@ -1,135 +1,25 @@
-Anaconda Recipes
-================
+**This repository is deprecated.**
 
-This is a Continuum managed repository for recipes used in the
-free [Anaconda Python distribution](https://www.continuum.io/downloads).
+Recipes used in the free 
+[Anaconda Python distribution](https://www.continuum.io/downloads) are hosted 
+in the Anaconda, Inc managed 
+[AnacondaRecipes/aggregate](https://github.com/AnacondaRecipes/aggregate)
+repository. 
 
-Building Conda Recipes
-================
+This repository holds recipes used in older versions of the distribution.
+The old README.md file for the repository has been renamed to [README_OLD.md](README_OLD.md).
 
+The feedstocks in the 
+[AnacondaRecipes](https://github.com/AnacondaRecipes/) organization, 
+are forked from <a href="https://github.com/conda-forge">conda-forge</a>,
+a community-driven recipe repo.
+Conda-forge is used as a reference for updating and fixing recipes in defaults.
+Community users encouraged to submit PRs to the conda-forge feedstock and then submit a PR to
+the AnacondaRecipes feedstock, showing that your changes work.
```

Comment:
I think let's ask people to submit changes at CF, then open an issue to ask us to pull their changes in.  We want to keep the git histories as mergeable as possible.

Reply:
Agreed, currently issues are turned off on AnacondaRecipes feedstocks.  Can we turn these on?

Reply:
I'm not an admin on the org.  I'll email IT with a pointer to this issue.